google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneGcpDialog.java

```diff
@@ -106,21 +108,21 @@ private void initComponents() {
     fcd.setDescription(GctBundle.message("clonefromgcp.destination.directory.description"));
     fcd.setHideIgnored(false);
     parentDirectory.addActionListener(new ComponentWithBrowseButton
-          .BrowseFolderActionListener<JTextField>(
-          fcd.getTitle(), fcd.getDescription(), parentDirectory,
-          project, fcd, TextComponentAccessor.TEXT_FIELD_WHOLE_TEXT) {
-        @Override
-        protected VirtualFile getInitialFile() {
-          String text = getComponentText();
-          if (text.length() == 0) {
-            VirtualFile file = project.getBaseDir();
-            if (file != null) {
-              return file;
-            }
-          }
-          return super.getInitialFile();
-        }
-      }
+                                          .BrowseFolderActionListener<JTextField>(
```

Comment:
Spacing doesn't seem right according to style guide. Did google-java-format do this?